index.js

```diff
@@ -34,15 +34,21 @@ Microcomponent.prototype.on = function (eventname, handler) {
   assert.equal(typeof handler, 'function', 'microcomponent.on handler should be type function')
 
   if (eventname === 'render') {
-    this._render = function (props) {
+    this._render = function () {
```

Comment:
can't we shorthand this to:
```js
var render = function () {
  return handler.call(this)
}
```

Or perhaps shorter if we skip declaring it here, and on line 47 we do:
```js
handler.call(this, props)
```
Or am I missing something?